services/web/server/src/simcore_service_webserver/projects/_tags_handlers.py

```diff
@@ -20,10 +20,12 @@
 routes = web.RouteTableDef()
 
 
-@routes.put(f"/{API_VTAG}/projects/{{project_uuid}}/tags/{{tag_id}}", name="add_tag")
+@routes.put(
+    f"/{API_VTAG}/projects/{{project_uuid}}/tags/{{tag_id}}", name="add_project_tag"
+)
 @login_required
 @permission_required("project.tag.*")
```

Comment:
why put and not post? this creates the tag? or you patch the project... or you replace all the tags on the project (which would then be put)

Reply:
@sanderegg we are passing a `tag_id`, meaning that you want to add a new tag. Note that it is not a `PUT` on the collection of project tags. 


That said, in reality here we are not creating a tag within a project but mainly **associating** an already existing `tag` to a `project`.  
Therefore what we are doing here is **creating the association/link** between these two resources.
Google-style action-based approach would therefore suggests

- `POST /projects/{project_id}/tags/{tag_id}:add` 
- `POST /projects/{project_id}/tags/{tag_id}:remove`